app/views/shared/_covid_banner.erb

```diff
@@ -2,12 +2,12 @@
 	<div class="l-constrained">
 		<div class="covid_banner__content">
 			<div class="covid_banner__inner">
-				<a href="#" class="covid_banner__close" data-dough-close>
+				<a href="#" class="covid_banner__close" aria-label=”close” data-dough-close>
```

Comment:
This value should  be available in both languages. It will exist elsewhere on the site. 
(Actually, our usual practice for labelling elements like this is to use text with a class of `visually-hidden`. In that case this would be the same a the mobile menu if that seems like a better option - whatever works).